packages/react-storage/src/components/StorageBrowser/views/hooks/usePaginate.ts

```diff
@@ -8,6 +8,7 @@ interface UsePaginate {
   }) => void;
   handlePaginatePrevious: (input?: {}) => void;
   handleReset: () => void;
+  range: [start: number, end: number];
 }
 
 export const usePaginate = ({
```

Comment:
Not sure if this was previously discussed but would make sense to return `pageItems` from `usePaginate` removing the need to expose `range`. Can be done in a follow up as we will need to update `usePaginate` to return the singular `onPaginate`

Reply:
Yeah, we discussed it. I'd love to get rid of range, but it'll have to be a follow up